src/main/webapp/user-private-comments.jsp

```diff
@@ -21,13 +21,13 @@
     <%if (currUser) {%>
       <h1 id="pc-header">Your Private Task Comments:</h1>
       <%for (TaskData task : tasks) {%>
-        
+        var jsData = <%=var jsData = (privateCommentsMap.get(task.getTaskID())).getMessage().trim()%>;
```

Comment:
update: looked up some stuff and found that I could declare as a var with this syntax. yet, this line is still erroring.